v3/src/lib/debug-event-modification.ts

```diff
@@ -0,0 +1,36 @@
+import { DEBUG_EVENT_MODIFICATION } from "./debug"
+
+const eventMethods = [
+  "preventDefault",
+  "stopPropagation",
+  "stopImmediatePropagation"
+] as const
+
+const elementMethods = [
+  "setPointerCapture"
```

Comment:
```suggestion
  "setPointerCapture",
  "releasePointerCapture"
```
I think it would be useful to track both to make it clear whether the capture was released or not.

Reply:
I'll add it, but I'd also note that it isn't necessary to call `releasePointerCapture`. According to https://developer.mozilla.org/en-US/docs/Web/API/Element/setPointerCapture the capture will be released automatically when the `pointerup` event is fired.

Reply:
Good to know.